api/dispatcher.proto

```diff
@@ -170,6 +170,23 @@ message AssignmentsRequest {
 	string session_id = 1 [(gogoproto.customname) = "SessionID"];
 }
 
+message Assignment {
+	oneof item {
+		Task task = 2;
+		Secret secret = 3;
```

Comment:
I'm not sure about this one.  :)  Can this be addressed in a future PR?  Seems like then an `AssignmentChange` should be one of `AssignmentUpdateChange` (which has an `Assignment` field) or `AssignmentRemoveChange` (which has a string field representing IDs).  Not sure if in the future we'd want to be able to remove by more than just ID, and that change might limit that ability.